python/lsst/pipe/tasks/hips.py

```diff
@@ -1619,33 +1592,70 @@ class GenerateColorHipsConfig(GenerateHipsConfig, pipelineConnections=GenerateCo
         doc="Band to use for blue channel of color pngs.",
         dtype=str,
         default="g",
+        deprecated=(
+            "This field is deprecated, set options in the rgbGenerator sub-config"
+            "This will be removed in version 29"
+        ),
```

Comment:
I think these need to say "removed after version 29", right?  Since you're planning to backport the deprecation but not the removal to v29?

Also, if these deprecated fields are now ignored, please include that in the deprecation messages.